src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -480,9 +480,9 @@ public Set<CompositeData> getCurrentStreams() {
   }
 
   @Override
-  public String getUntranslatedHost() {
-    //TODO: implement me
-    return "";
+  public String getUntranslatedHost() throws ReaperException {
+    // TODO: we eventually need to implement address translation.
+    return getLocalEndpoint();
```

Comment:
**todo:** We're missing a unit test for this method. It will obviously be very simple but will check that the method returns whatever we mock into the getLocalEndpoint() method.

Reply:
My bad there, it was such a basic one that I didn't even think of it. Bad Miles!